test/intl402/Locale/prototype/textInfo/output-object-keys.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.locale.prototype.textInfo
+description: >
+    Checks that the return value of Intl.Locale.prototype.textInfo is an Object
+    with the correct keys and properties.
+info: |
+  get Intl.Locale.prototype.textInfo
+  ...
+  7. Perform ! CreateDataPropertyOrThrow(info, "direction", dir).
+  features: [Intl.Locale,Intl.Locale-info]
```

Comment:
"features:" need to start with the line,  please remove two SPACE before that, otherwise, it will be treated as part of the "info:"